gradle.properties

```diff
@@ -1,25 +1,25 @@
 org.gradle.jvmargs=-Xmx3G
 org.gradle.daemon=false
 
-minecraft_version=1.20.5
-supported_version=1.20.5
+minecraft_version=1.21-rc1
+supported_version=1.21-rc1
 
 archives_base_name=cloth-config
 archives_base_name_snapshot=cloth-config-snapshot
-base_version=14.0
+base_version=15.0
 maven_group=me.shedaniel.cloth
 
 jankson_version=1.2.0
 toml4j_version=0.7.2
 snakeyaml_version=1.27
 
-fabric_loader_version=0.15.10
-fabric_api_version=0.97.6+1.20.5
-mod_menu_version=10.0.0-beta.1
+fabric_loader_version=0.15.11
+fabric_api_version=0.100.1+1.21
+mod_menu_version=11.0.0-beta.1
 
 forge_version=49.0.3
-neoforge_version=20.5.0-beta
-neoforge_pr=
+neoforge_version=21.0.0-alpha.1.21-rc1.20240613.054156
+neoforge_pr=1076
```

Comment:
Release is now available: https://github.com/neoforged/NeoForge/commit/dc521863d377a8be232488e6a57dd1eb758928f6

Reply:
Thank you for notifying me!